src/sql/src/parsers/tql_parser.rs

```diff
@@ -41,23 +94,31 @@ impl<'a> ParserContext<'a> {
         match self.parser.peek_token().token {
             Token::Word(w) => {
                 let uppercase = w.value.to_uppercase();
+                let _consume_tql_keyword_token = self.parser.next_token();
                 match w.keyword {
                     Keyword::NoKeyword
                         if (uppercase == EVAL || uppercase == EVALUATE)
                             && w.quote_style.is_none() =>
                     {
-                        let _ = self.parser.next_token();
                         self.parse_tql_eval().context(error::SyntaxSnafu)
```

Comment:
@waynexia 
provided that`parse_tql_eval`, `parse_tql_analyze` and `parse_tql_explain` essentially became a wrapper of `parse_tql_params` does it make sense to do something like
```suggestion
                            self.parse_tql_params()
                                .map(|params|Statement::Tql(Tql::Eval(TqlEval::from(params))))
                                .context(error::SyntaxSnafu)
```
WDYT?